mhs/common/mhs_common/messages/ebxml_error_envelope.py

```diff
@@ -0,0 +1,44 @@
+from typing import Dict, Tuple, List, AnyStr, Any
+from xml.etree import ElementTree as ET
+
+from mhs_common.messages.ebxml_envelope import EbxmlEnvelope, NAMESPACES as NS, EBXML_NAMESPACE as EBXML_PREFIX
+from mhs_common.messages.envelope import Envelope
+
+
+ERROR_LIST = 'ErrorList'
+ERROR = 'Error'
+ERR_CODE_CTX = 'codeContext'
+ERR_CODE = 'errorCode'
+ERR_SEVERITY = 'severity'
+ERR_DESC = 'Description'
+
+
+class EbxmlErrorEnvelope(EbxmlEnvelope):
+    def __init__(self, errors: List[Dict[AnyStr, Any]]):
```

Comment:
Looking at the code, it looks like this can be made more specific as:
```suggestion
    def __init__(self, errors: List[Dict[AnyStr, str]]):
```

Reply:
Changed.